.pre-commit-config.yaml

```diff
@@ -43,5 +43,4 @@ repos:
     hooks:
       - id: black
         name: Format code
-        args: [--skip-string-normalization, --line-length=119]
```

Comment:
Hmm can we keep this here as well as in toml?

Reply:
Yes, but I don't think that such duplication is useful

Reply:
`isort` options are also used from `pyproject.toml` here